docs/physics/tardisgamma/decayenergy.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import numpy as np
</pre><p>Stylistically, I'd have the import statement in a separate cell from any content, at the top of the notebook. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
I included the imports in a separate cell